tests/kit_test.cpp

```
@@ -0,0 +1,61 @@
+// The generated lofi header carries PRD Appendix A exactly (D-027), so the engine
+// tests run against the real kit data rather than a fixture.
+#include <cstring>
+
+#include "engine_support.h"
+
+using namespace support;
+
+TEST_CASE("lofi kit header matches PRD Appendix A") {
```

Clang (14.0.6)

[warning] variable 'TEST_CASE' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)